tools/tasks/seed/app.through.ts

```diff
@@ -0,0 +1 @@
+import '../../../src/client/app/main';
```

Comment:
Thanks! I added this as [last step](https://github.com/mgechev/angular-seed/wiki/Enabling-Angular-i18n-native) in order to not pollute the seed's tasks directory.